src/py42/services/cases.py

```diff
@@ -0,0 +1,207 @@
+from py42 import settings
+from py42.exceptions import Py42BadRequestError
+from py42.exceptions import Py42InvalidActionError
+from py42.services import BaseService
+from py42.services.util import get_all_pages
+
+
+class CasesService(BaseService):
+
+    _uri_prefix = u"/api/v1/case"
+
+    def __init__(self, connection):
+        super(CasesService, self).__init__(connection)
+
+    def create(
+        self, name, subject=None, assignee=None, description=None, findings=None
+    ):
+        """Creates a new case.
+
+        Args:
+            name (str): Name of the case.
+            subject (str, optional): User UID of the subject of the case.
+            assignee (str, optional): User UID of the assignee.
+            description (str, optional): Description of the case.
+            findings (str, optional): Notes on the case.
+
+        Returns
+            :class:`py42.response.Py42Response`
+        """
+        data = {
+            u"assignee": assignee,
+            u"description": description,
+            u"findings": findings,
+            u"name": name,
+            u"subject": subject,
+        }
+        return self._connection.post(self._uri_prefix, json=data)
+
+    def get_page(
+        self,
+        name=None,
+        status=None,
+        created_at=None,
+        updated_at=None,
+        subject=None,
+        assignee=None,
+        page_num=1,
+        page_size=None,
+        sort_direction=u"asc",
+        sort_key=u"number",
+        **kwargs
+    ):
+        """Gets an individual page of cases.
+
+        Args:
+            name (str, optional): Filter results by case name, matches partial names. Defaults to None.
+            status (str, optional): Filter results by case status, `OPEN` or `CLOSED`. Defaults to None.
+            created_at (str, optional): Filter results by case creation time range, format ISO interval.
+                Defaults to None. e.g 2020-08-31T11:00:00Z/2020-09-01T15:30:00Z
```

Comment:
`Defaults to None.` should be the closing part of the parameter doc, not in the middle

Reply:
This happens in other spots as well

Reply:
Actually.... Might even be preferred to just remove all the docs from this service since this is not a public service. That way, less stuff to manage and make sure is correct

Reply:
agree here, only the client part that the users actually interact with need docstrings.

Reply:
Removed docs from service.